includes/class-sensei-wc.php

```diff
@@ -1233,8 +1233,12 @@ public static function complete_order( $order_id = 0 ) {
 		    return;
         }
 		$order_status = Sensei_WC_Utils::get_order_status( $order );
+		
+		$accepted_order_stati = array( 'wc-completed', 'wc-processing' );
+		
+		$additional_order_stati = apply_filters( 'sensei_wc_add_order_stati', $accepted_order_stati );
```

Comment:
`stati` is not the plural form of statuses. Could we rename the filter to `sensei_wc_order_statuses` to better fit with how both Sensei and WooCommerce name their filters?

Also, it would be nice to document the filter as per the [PHP documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters).